src/ui-test/utils.ts

```diff
@@ -484,3 +491,92 @@ export async function selectTask(driver: WebDriver, task: string): Promise<void>
     }
 }
 
+/**
+ * Reset user setting to default value by deleting item in settings.json.
+ *
+ * @param id ID of setting to reset.
+ */
+export function resetUserSettings(id: string): void {
+    const settingsPath = path.resolve(storageFolder, 'settings', 'User', 'settings.json');
+    const reset = fs.readFileSync(settingsPath, 'utf-8').replace(new RegExp(`"${id}.*`), '').replace(/,(?=[^,]*$)/, '');
+    fs.writeFileSync(settingsPath, reset, 'utf-8');
+}
+
+/**
+ * Read user settings value directly from settings.json
+ *
+ * @param id ID of setting to read.
+ */
+export function readUserSetting(id: string): string | null {
+    const settingsPath = path.resolve(storageFolder, 'settings', 'User', 'settings.json');
+    const settingsContent = fs.readFileSync(settingsPath, 'utf-8');
+
+    const regex = new RegExp(`"${id}":\\s*"(.*?)"`, 'i');
+    const match = settingsContent.match(regex);
+
+    if (match === null) {
+        return null;
+    } else {
+        return match[1];
+    }
+}
+
+/**
+ * Set user setting directly inside settings.json
+ * 
+ * @param id ID of setting.
+ * @param value Value of setting.
+ */
+export function setUserSettingsDirectly(id: string, value: string): void {
+    const settingsPath = path.resolve(storageFolder, 'settings', 'User', 'settings.json');
+    const settings = JSON.parse(fs.readFileSync(settingsPath, 'utf-8'));
+    settings[id] = value;
+    fs.writeFileSync(settingsPath, JSON.stringify(settings, null, 4), 'utf-8');
+}
+
+/**
+ * Checks if given Camel version is productized.
+ * 
+ * @param input Camel version as string.
+ * @returns true/false
+ */
+export function isCamelVersionProductized(input: string): boolean {
+    const pattern = /\.redhat-\d+$/;
+    return pattern.test(input);
+}
+
+/**
+ * Extract Camel version number. Productized or non-productized version can be provided as input.
+ * 
+ * @param input Camel version in format "x.x.x.redhat-xxx" or "x.x.x".
+ * @returns Camel version in format "x.x.x".
+ */
+export function extractVersionNumber(input: string): string {
+    if (isCamelVersionProductized(input)) {
+        const regex = /^(.*?)\.[^.]*$/;
+        const match = regex.exec(input);
+        return match ? match[1] : '';
+    } else {
+        return input;
+    }
+}
+
+/**
+ * Compare two versions in format "^\d+(\.\d+)*$".
+ * @param base Base version.
+ * @param target Version to be compared with base version.
+ * @returns true if target is newer or same as base, false otherwise
+ */
+export function compareVersions(base: string, target: string): boolean {
```

Comment:
I think we can find a more precise naming, somethign like `isVersionNewer` or `isVersionHigher`

Reply:
You're right. Changed.